### DISCUSSION TITLE
Howto bridge eth0 with wlan?

### DISCUSSION THREAD

**Discussion by rfhenke69** [RaspAP/raspap-webgui#816] [Q&A]
I'm using my setup with my android phone via USB and this works great for as it should as Wifi hot-spot, but I have a Ethernet only device I was hoping to plug into my raspberry's eth0 port. I kinda thought this would work automatic, but it didn't and I also can't find a config setting...

I did manage using the interface to switch eth0 from DHCP to static and enabled a DHCP server with a new range and this worked to provide my device on eth0 with a IP via DHCP from that range... but traffic is not routed out via USB...

I don't mind having a different DHCP range on eth0 and WLAN, but I guess simply bridging eth0 and WLAN would be the perfect solution.

Any advise?

**Comment by thierryR51** [comment]
Hi, this is a suggestion.
You can connect an adapter usb3/rj45 . it's directly recognized.      https://www.amazon.fr/gp/product/B00MYTSN18/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1
```
ifconfig
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.8.26  netmask 255.255.255.0  broadcast 192.168.8.255
        inet6 fd51:42f8:caae:d92e::ff  prefixlen 64  scopeid 0x0<global>
        inet6 fd1c:b796:63f2:b500:aea7:8862:cd3f:5b37  prefixlen 64  scopeid 0x0<global>
        inet6 fe80::e491:bfa4:ed39:212c  prefixlen 64  scopeid 0x20<link>
        ether e4:5f:01:03:a6:db  txqueuelen 1000  (Ethernet)
        RX packets 71958  bytes 55292952 (52.7 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 65446  bytes 13664210 (13.0 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth1: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        ether 00:0e:c6:70:68:1c  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
```
The difficulty remains in the configuration.

**Reply by rfhenke69** [reply]
thanks, but I have no clue how this would solve my problem? I don't need a 2nd ethernet adapter

I just want traffic from my build-in ethernet eth0 been routed via my cell (usb) the same way wifi traffic is routed out...

Ether I assume a routing issue or by bridging wifi and ethernet... which I tried but failed
